src/com/rcv/Tabulator.java

```diff
@@ -24,28 +26,53 @@
     SKIP_TO_NEXT_RANK,
   }
 
+  enum TiebreakMode {
+    RANDOM,
+    INTERACTIVE,
+    PREVIOUS_ROUNDS_THEN_RANDOM,
+    PREVIOUS_ROUNDS_THEN_INTERACTIVE,
```

Comment:
Maybe PREVIOUS_ROUND_TALLY_THEN_RANDOM, PREVIOUS_ROUND_TALLY_THEN_INTERACTIVE (or count instead of tally)